terminus-ui/scss/helpers/test-sass.md

```diff
@@ -76,12 +78,12 @@ In this scenario the assert has two pieces, an output that includes the mixin, a
 
 ### Test to Fail
 
-It's highly recommended to write your tests to fail, to insure the test is running properly. In other words, it gives a lot of false-positives. Once you know the test is running correctly, then change the expect value to be the correct one.
+It's highly recommended to write your tests to fail, to insure the test is running properly. In other words, it gives a lot of
+false-positives. Once you know the test is running correctly, then change the expect value to be the correct one.
 
 
 ## Issues, idiosyncracies, etc
 
-1. Not quite sure how to navigate large blocks of rendered scss (reference _color.spec.scss’s commented-out test. Currently investigating how `true` adds classes in the tests so we can better mitigate this critical functionality.
-
+1. Not quite sure how to navigate large blocks of rendered scss (reference `color.spec.scss`s commented-out test. Currently investigating
+   how `true` adds classes in the tests so we can better mitigate this critical functionality.
```

Comment:
Same here and line 82

Reply:
Yeah, these were simply too long. Not sure how/why the linter didn't catch it